addons/mass_mailing/views/mass_mailing.xml

```diff
@@ -448,7 +448,10 @@
 
                         <group>
                             <field name="email_from"/>
-                            <field name="name"/>
+                            <label for="id" string="Subject"/>
```

Comment:
Because it's taking the Name from the inheritS source_id, which is 'Source Name'.